config/local_devnet/genesis_config.json

```diff
@@ -5,8 +5,12 @@
     "l2GenesisBlockDifficulty": "0x0",
     "l2GenesisBlockNumber": "0x0",
     "l2GenesisBlockGasUsed": "0x0",
-    "l2GenesisBlockBaseFeePerGas": "0x0",
+    "l2GenesisBlockBaseFeePerGas": "0x1",
     "l2GenesisBlockExtraData": "0x",
     "l2PredeployOwner": "0x000000000000000000000000000000000000dEaD",
+    "l2FeesWithdrawalAddress": "0x000000000000000000000000000000000000dEaD",
```

Comment:
does having this in the genesis mean we can’t change the withdrawal address without a hard fork?

Reply:
yeah (also this value is ofc temporary, we need to set it properly)